lib/transform.js

```diff
@@ -136,8 +114,7 @@ exports = module.exports = function transform(content, mode, maxHeaderLevel, tit
   var currentToc = info.hasStart && lines.slice(info.startIdx, info.endIdx).join('\n')
     , linesToToc = getLinesToToc(lines, currentToc, info);
 
-  var headers = getHashedHeaders(linesToToc, maxHeaderLevel)
-    .concat(getUnderlinedHeaders(linesToToc, maxHeaderLevel))
+  var headers = getMarkdownHeaders(linesToToc, maxHeaderLevel)
     .concat(getHtmlHeaders(linesToToc, maxHeaderLevelHtml))
```

Comment:
I just wanted to get the PR out so you could take a look at it. `markdown-to-ast` will give ast nodes to HTML, but it won't parse it.